src/macaron/config/defaults.ini

```diff
@@ -600,3 +600,17 @@ major_threshold = 20
 epoch_threshold = 3
 # The number of days +/- the day of publish the calendar versioning day may be.
 day_publish_error = 4
+
+# disable default semgrep rulesets here (i.e. all rule IDs in a Semgrep .yaml file) using comma separated ruleset names. Use the name
+# without the .yaml prefix here. Currently, we disable the exfiltration rulesets by default due to a high false positive rate.
+disabled_default_rulesets = exfiltration
```

Comment:
Could you please walk me through the differences between the default rules and the individual rules ? Are they from two separate set, and doesn't have any interaction with each other ?